package.json

```diff
@@ -3,12 +3,14 @@
   "version": "0.2.3",
   "description": "GameChanger Metrics API",
   "main": "lib/metrics.js",
+  "types": "lib/metrics.d.ts",
   "author": "Tom Leach",
   "dependencies": {
-    "@gamechanger/config": "^0.1.3",
+    "@gamechanger/config": "git+ssh://git@github.com:gamechanger/gcconfig.js.git#0.1.3",
```

Comment:
This was weirdly inconsistent, this is a public repo but it was pointing at a package in our private reg. The private reg is sort of a pain in the ass and I'm not sure what value it provides over GitHub. Plan is to make this repo private and then use GitHub rather than private reg in all instances we need this going forward.